cmake/Modules/Platform/Emscripten.cmake

```
@@ -102,6 +107,12 @@ if ("${CMAKE_RANLIB}" STREQUAL "")
   set(CMAKE_RANLIB "${EMSCRIPTEN_ROOT_PATH}/emranlib${EMCC_SUFFIX}" CACHE FILEPATH "Emscripten ranlib")
 endif()
 
+if ("${CMAKE_STRIP}" STREQUAL "")
+  set(CMAKE_STRIP "${EMSCRIPTEN_ROOT_PATH}/llvm-strip" CACHE FILEPATH "Emscripten compatible strip command"))
```

Not sure about that path. On my local machine, its more like:
```suggestion
  set(CMAKE_STRIP "${EMSCRIPTEN_ROOT_PATH}/../bin/llvm-strip" CACHE FILEPATH "Emscripten compatible strip command"))
```

---

@sbc100 do you know where `llvm-trip` should be found for an emscripten installation with respect to `EMSCRIPTEN_ROOT_PATH`?

We can probably also write "llvm-trip" as a value without a path, risking that it picks up another version of `llvm-trip` from system paths.

---

Yes, you are correct, its non-trivial to get the correct llvm-strip here.  I guess we should have an `emstrip` wrapper script.

In any case it seems unrelated to this PR.. can you create a separate issue for that?

---

@ax3l you have an extra parentheses breaking some of the tests

---

Thanks, fixed now.

---

@sbc100:
> Yes, you are correct, its non-trivial to get the correct llvm-strip here. I guess we should have an emstrip wrapper script.
>
> In any case it seems unrelated to this PR.. can you create a separate issue for that?

Sounds good, opened as #16430.

---

Will update now to use `emstrip` - thanks #16431 :sparkling_heart: